Source/Stash.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import Security
+
+public struct Stash {
+
+  enum Action {
+    case Insert, Fetch, Delete
+  }
+
+  // MARK: - Public methods
+
+  public static func password(service: String, account: String) -> String {
+    guard !service.isEmpty && !account.isEmpty else { return "" }
+
+    let query = [
+      kSecAttrAccount as String : account,
+      kSecAttrService as String : service,
+      kSecClass as String : kSecClassGenericPassword]
+
+    return Stash.query(.Fetch, query).1
+  }
+
+  public static func setPassword(password: String, service: String, account: String) -> Bool {
```

Comment:
No I was thinking to return an optional Bool. Just in case you don't want to set a return value. You know what I mean? Or I think I'm being crazy now. You don't have to assign it right even if you return something? If so, then nevermind. :smile: I just woke up when I commented...